src/v2/view-builder/views/webauthn/ChallengeWebauthnView.js

```diff
@@ -68,7 +73,7 @@ const Body = BaseForm.extend({
     const relatesToObject = this.options.currentViewState.relatesTo;
     const authenticatorData = relatesToObject?.value || {};
     const allowCredentials = [];
-    const authenticatorEnrollments = this.options.appState.get('authenticatorEnrollments').value || [];
+    const authenticatorEnrollments = this.options.appState.get('authenticatorEnrollments')?.value || [];
```

Comment:
Sorry, I'm not super familiar with js. What exactly does this diff do?

Reply:
Hi @mpeng-okta , for the residentkey flow, we do not have the `authenticatorEnrollments` object, so it will throw an exception without using this `optional chaining operator (?.)`

Reply:
Got it, basically it would be missing entirely. Yeah, that makes sense.